### DISCUSSION TITLE
fed my TS80P with 20V istead of 12V

### DISCUSSION THREAD

**Discussion by discip** [Ralim/IronOS#1482] [Q&A]
Unfortunately I did what is stated in the header.

Since I can't remember what values the Iron displayed prior to overcharging it, I need someones help at figuring out if I should get replacement parts for it, or if everything is still fine.

I am reading:
||value
-|-:
Tip Res| 4.5
Tip R|1452
Tip O|1554

What values does your **`TS80P`** give you with the **`TS-B02`** tip after calibration?

Thanks in advance
kind regards

**Comment by KoopsF** [comment]
Where can I find these values? I found RTip and CTip in the detailed menu by holding the second button.

**Reply by discip** [reply]
Good morning,
first of all: Thank you for trying to help me. 😊
Yes that's the right place to look for, but you are using an 'older' version of IronOS where **`Tip O`** is not present.

If you don't mind, you could flash this one: [latest](https://github.com/Ralim/IronOS/suites/9514936904/artifacts/451752843)
Please follow this guide to calibrate the iron: [Calibrate CJC](https://github.com/Ralim/IronOS/blob/dev/Documentation/Menu.md)

thanks in advance

**Reply by KoopsF** [reply]
alright, the values I get are:
Tip Res = 4.5
Tip R = jumps around 1650
Tip O =1431

**Reply by discip** [reply]
Thank you!
Did you use the **`TS-B02`** tip?

**Reply by KoopsF** [reply]
Yes

**Comment by discip** [comment]
@Ralim
Can you please help me figure out how to check which component is broken?

**Reply by Ralim** [reply]
Your Tip R and Tip O values look _fairly_ normal.
Is it misbehaving?

**Reply by discip** [reply]
Yes it is.
Prior to this incident the external measured temperature was almost spot on (never above the desired temp), whereas after that it does overshoot iirc by almost 10°C.

**Reply by Ralim** [reply]
Is it measuring correctly at rest?
If its just overshooting at higher temps it could be tip damage or the op-amp.

**Reply by discip** [reply]
I'll have to wait for a hopefully more reliable meter to arrive.
Will report as soon as possible.
